.travis.yml

```diff
@@ -9,6 +9,9 @@ env:
   - DJANGO='django<1.5'
   - DJANGO='django<1.6'
   - DJANGO='django<1.7'
+  - DJANGO='django<1.8'
+  - DJANGO='django<1.9'
```

Comment:
`django<1.9` is intended to install django 1.8.x and not the beta.

My intention with `django<1.10` was to start testing django 1.9 as soon as it is released.  But I can pin the version to 1.9b1 if you like this better.